src/lib.rs

```diff
@@ -161,6 +161,24 @@ impl NotionApi {
         }
     }
 
+    /// Get a page by [PageId].
+    pub async fn get_page<T: AsIdentifier<PageId>>(
```

Comment:
Would it be possible to have a test covering this new method?

Reply:
I will work on it. Could you please fix existing tests to work on github, they are failing because of the auth issues?

Reply:
Ahh, I see. I thought all the tests were cut over to Wiremock. I can address that.